src/index.js

```diff
@@ -170,16 +170,32 @@ window.kinoInitGoogleCast = (function kinoInitGoogleCastIIFE() {
   return () => {
     if (!castButtonPromise) {
       castButtonPromise = new Promise((resolve) => {
+        const castButton = document.createElement('button');
+        const castCustomElement = document.createElement('google-cast-launcher');
+
+        castButton.setAttribute('aria-label', 'Cast this video');
+        castButton.appendChild(castCustomElement);
+        castButton.classList.add(CAST_BUTTON_HIDDEN_CLASSNAME);
```

Comment:
This line is not needed as `castButton` will only be added to the document once promise is resolved right?

Reply:
@beaufortfrancois The button is added to the document as soon as the Google Cast API becomes available after the recent changes – regardless of whether cast devices are available or not.

The button visibility (controlled using the class name) then depends on cast state changes.

Reply:
Quite right. 🤦‍♂️

Thank you. Removed in 3cc1ef6.